SPEC.md

```diff
@@ -436,187 +436,37 @@ The Data Connect service responds with the following table:
 
 ## SQL Functions
 
-Data Connect specification is implementation-agnostic and does not prescribe use of a relational database or a particular database technology. However, for convenience, its SQL dialect has been selected for compatibility with current major open source database platforms including Trino, PostgreSQL, MySQL, and well as BigQuery. There are occasional name or signature differences, but a Data Connect API implementation atop any of the major database platforms should be able to pass through queries that use the functions listed below with only minor tweaks.
+The Data Connect specification is implementation-agnostic and does not prescribe use of a relational database or a particular database technology. As a result, the exact SQL dialect that is available will vary between implementations of the standard.
 
-With Trino (formerly PrestoSQL) being a popular database choice in Data Connect implementations, we've chosen its grammar as the basis for the grammar supported by Data Connect. Functions below are a subset of those available in Trino 341, and must behave according to the Trino documentation in a conforming Data Connect implementation. To assist with implementations directly on other database platforms, the [Trino Functions Support Matrix](https://docs.google.com/document/d/1y51qNuoe2ELX9kCOyQbFB4jihiKt2N8Qcd6-zzadIvk) captures the differences between platforms in granular detail. 
+The functions listed below SHOULD be supported by any implementation of Data Connect that supports the `search` endpoint.  These functions are supported by major database platforms including Trino, PostgreSQL, MySQL and BigQuery. There are occasional name or signature differences, but a Data Connect API implementation atop any of the major database platforms should be able to pass through queries that use the functions listed below with only minor tweaks.
 
+*   ga4gh_type (described above)
 *   **Logical Operators**
     *   `AND`, `OR`, `NOT`
 *   **Comparison Operators**
     *   `&lt;`, `>`, `&lt;=`, `>=`, `=`, `&lt;>`, `!=`
```

Comment:
I've just noticed `<` is not escaped correctly.

Reply:
I changed this just to `<` since it shouldn't be escaped at all AFAIK.